lib/mpi/examples/src/mpi_simple.nit

```diff
@@ -27,8 +27,7 @@ class ProcessorInfo
 	var name: String
 	var string_of_random_length: String
 
-	init(mpi: MPI)
-	do
+	init(mpi: MPI) is old_style_init do
```

Comment:
Instead of adding the annotation, the constructor should be updated. And come on, don't change the style.